tests/tt_metal/tt_metal/unit_tests/basic/runtime_args.cpp

```diff
@@ -128,7 +128,7 @@ bool verify_results(
 
         // Verify common RT Args (same for all cores) if they exist.
         if (common_rt_args.size() > 0) {
-            const auto common_args_addr = rt_args_base_addr + common_rt_args_offset; // Common args are placed after unique args per core.
+            const auto common_args_addr = rt_args_base_addr + align(common_rt_args_offset, 16); // Common args are placed after unique args per core. 16B aligned.
```

Comment:
use NOC_L1_ALIGNMENT

Reply:
We want to remove L1_ALIGNMENT and PCIE_ALIGNMENT. They're duplicates of `NOC_` versions.